src/context.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import * as Emitter from 'tiny-emitter';
+
+interface PortalProviderProps {}
+
+export interface PortalValue {
+  subscribeToPortal: Function;
+  unsubscribeFromPortal: Function;
+  setPortal: Function;
+  getPortal: Function;
```

Comment:
I think we have a linter rule set up to disallow this and I think we should follow it here:

Use the explicit `Function` type via arrow syntax:

```suggestion
  getPortal: (name: string) => any[] | undefined;
```
(and for the rest of them, too)

We'll want these explicit types when we're actually using these.

Reply:
0f806d0